src/codegen/doc/mod.rs

```diff
@@ -46,41 +47,49 @@ pub fn generate(env: &Env) {
 
 fn generate_doc(mut w: &mut Write, env: &Env) -> Result<()> {
     try!(write_file_name(w, None));
+    let mut generators: Vec<(&str, Box<Fn(&mut Write, &Env) -> Result<()>>)> = Vec::new();
+
+    let ignored = |info: &InfoBase| {
+        info.type_id.ns_id != MAIN ||
+            env.config.objects.get(&info.full_name).map_or(true, |obj| obj.status.ignored())
```

Comment:
I'm assuming we don't want to emit docs for items which are ignored and thus absent from the crate.